app.js

```diff
@@ -1 +1,165 @@
+// 10
+new FocusImage('utensil pen caps', 'img/pen.jpg', 'pen-img', allImages);
+// 11
+new FocusImage('pet footie sweepers', 'img/pet-sweep.jpg', 'pet-sweep-img', allImages);
+// 12
+new FocusImage('pizza scissors', 'img/scissors.jpg', 'scissors-img', allImages);
+// 13
+new FocusImage('shark sleeping bag', 'img/shark.jpg', 'shark-img', allImages);
+// 14
+new FocusImage('baby onesie sweeper', 'img/sweep.png', 'sweep-img', allImages);
+// 15
+new FocusImage('tauntaun sleeping bag', 'img/tauntaun.jpg', 'tauntaun-img', allImages);
+// 16
+new FocusImage('unicorn meat', 'img/unicorn.jpg', 'unicorn-img', allImages);
+// 17
+new FocusImage('tentacle usb', 'img/usb.gif', 'usb-img', allImages);
+// 18
+new FocusImage('self-watering can', 'img/water-can.jpg', 'water-can-img', allImages);
+// 1
+new FocusImage('closed-top wine glass', 'img/wine-glass.jpg', 'wine-glass-img', allImages);
+
+/**
+creates an object for a image with the name, path, and id. adds to given array.
+**/
+function FocusImage(imageTitle, imagePath, imageId, imageArray) {
+  this.imageTitle = imageTitle;
+  this.imagePath = imagePath;
+  this.imageId = imageId;
+  this.timesClick = 0;
+  this.timesShow = 0;
+  imageArray.push(this);
+}
+
+/**
+chooses a random index from an array
+**/
+function randomIndex(imageArray) {
+  return Math.floor(Math.random() * imageArray.length);
+}
+
+/**
+picks 3 images and returns them, no duplicates. ignores images from the previous round.
+**/
+function randomImages(remainingImages, previousImages) {
+  var chosenIndex;
+  var chosenImages = [];
+  var numToChoose = 3;
+  for (var i = 0; i < numToChoose; i++) {
+    chosenIndex = randomIndex(remainingImages);
+    chosenImages.push(remainingImages[chosenIndex]);
+    remainingImages.splice(chosenIndex, 1); // removes chosen item
+  }
+  for (var j = 0; j < previousImages.length; j++) {
+    remainingImages.push(previousImages[j]); // add back in the previous images
+  }
+  return chosenImages;
+}
+
+/**
+sticks the chosen images in a div to print. adds event listeners to each.
+**/
+function renderImages(chosenImages) {
+  var div = document.createElement('div');
+  div.id = 'app-images';
+  var image;
+  var printImage;
+  for (var i = 0; i < chosenImages.length; i++) {
+    printImage = chosenImages[i];
+    image = document.createElement('img');
+    image.src = printImage.imagePath; //display the image
+    image.id = printImage.imageId;
+    printImage.timesShow++; //increment times shown
+    image.addEventListener('click', handleImageClick);
+    div.appendChild(image);
+  }
+  return div;
+}
+
+/**
+handles a click on one of the set of images. gives another set if more rounds required, removes the listener if done.
+**/
+function handleImageClick(e) {
+  console.log(currentRound);
+  var imageId = e.target.id;
+  var imageObj;
+  for (var i = 0; i < allImages.length; i++) {
+    if (imageId === allImages[i].imageId) {
+      imageObj = allImages[i];
+      i = allImages.length;
+    }
+  }
```

Comment:
This can be refactored to not need a loop at all.  Consider using your target.id as an index value for either a displayed image array or all product array